GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -84,4 +84,24 @@ extension Glia {
             sceneProvider: sceneProvider
         )
     }
+
+    /// Deprecated, use `clearVisitorSessionIfPossible() throws` instead.
+    @available(
+        *,
+         deprecated,
+         message: "Use clearVisitorSessionIfPossible() throws instead."
+    )
+    public func clearVisitorSession() {
```

Comment:
But isn't this changing the previous version? With the previous one, the clearing was executed always, with whatever side effects it would had. With this new version, it only is executed if it's possible, otherwise it fails silently. While this is clearly better, it is still modifying the old one which I think we are not allowed to do.

Reply:
I will raise discussion.

Reply:
Conclusion of the [discussion](https://salemove.slack.com/archives/C01F9K88SQZ/p1684504908521729):
`add to existing old method the error logging (into debug console) about active engagement. Existing old method will work as previously, but with logging error to debug console.`

c4dbce994198b4b072929de2e4133ec8f0df2e60